docs/core/porting/upgrade-assistant-install-legacy.md

```diff
@@ -0,0 +1,164 @@
+---
+title: How to use the legacy .NET Upgrade Assistant
+description: This article describes how to install the old .NET Upgrade Assistant tool.
+author: adegeo
+topic: how-to
+ms.date: 05/22/2023
+---
+# Install the .NET Upgrade Assistant (legacy)
```

Comment:
Should we call this article something like "Legacy version of Upgrade Assistant" not the "Install the .NET Upgrade Assistant (legacy)" since it covers not just installation but also the usage?